renderer/components/Onboarding/Steps/ConnectionDetailsManual.js

```diff
@@ -32,6 +34,7 @@ class ConnectionDetailsManual extends React.Component {
   static defaultProps = {
     wizardApi: {},
     wizardState: {},
+    name: '',
```

Comment:
why default to an empty string rather than null?

Reply:
Done.